packages/legacy/cli/src/index.ts

```diff
@@ -22,6 +25,8 @@ import { generateTsArtifacts } from './commands/ts-artifacts.js';
 import { findAndParseConfig } from './config.js';
 import { handleFatalError } from './handleFatalError.js';
 
+register('ts-node/esm', pathToFileURL('./'));
```

Comment:
Does the path here need to respect `baseDir` or `cwd` given by the user?

Reply:
Seems to be working well as-is :)